otsdaq-mu2e/ROCCore/ROCCoreVInterface.h

```diff
@@ -108,6 +109,8 @@ class ROCCoreVInterface : public FEVInterface
 	//----------------- Emulator members
 	// return false when done with workLoop
   public:
+  	virtual void       					    GetStatus  					(__ARGS__);  
```

Comment:
I think we should just make these functions 'pure virtual' which means in the .h they are declared = 0. Then all inheriting classes must define. No need for code in the .cc